Ontology/Asset/Equipment/ICTEquipment/ControllerEquipment/BACnetController.json

```diff
@@ -0,0 +1,23 @@
+{
+    "@id": "dtmi:digitaltwins:rec_3_3:asset:BACnetController;1",
+    "@type": "Interface",
+    "displayName": {
+      "en": "BACnet Controller"
+    },
+    "extends" : [
+      "dtmi:digitaltwins:rec_3_3:asset:ControllerEquipment;1"
+    ],
+    "contents": [
+      {
+        "@type": "Property",
```

Comment:
I would remove this `address` property as it is a Willow specific implementation detail.

Reply:
Done